packages/commercetools/composables/src/getters/_utils.ts

```diff
@@ -84,7 +84,7 @@ const getSpecialPrice = (product: ProductVariant | LineItem) => {
 };
 
 export const createPrice = (product: ProductVariant | LineItem): AgnosticPrice => {
-  if (!product) {
+  if (!product || product.lineItemMode === 'GiftLineItem') {
```

Comment:
I think you could use the isLineItem(product) function instead of extending the ProductVariant with lineItemMode (this might be confusing as the commercetools ProductVariant does not have the attribute)